extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -36,7 +35,6 @@ import {toggle} from '../../../src/style';
 installActivityService(AMP.win);
 installCidService(AMP.win);
 installCryptoService(AMP.win);
-installStorageService(AMP.win);
```

Comment:
I don't fully see it, how can we get to a case where `v0.js` and an extension are different versions? (at least for now that everything is 0.1)